FreeRTOS/FreeRTOS
# Fix license double spacing issue

Description
-----------
The line 'Copyright (C) 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.' used to have two spaces between the first and second sentences. This would cause the header check to fail due to the copyright regex located
https://github.com/FreeRTOS/FreeRTOS/blob/ab999f9624ec750f8cb153d074ccc0681d030142/.github/scripts/core_checker.py#L396.

Getting tired of PRs failing checks because of this header spacing problem.

To create this I simply did a find+replace all on the broken license string for all `.c` and `.h` files.

Test Steps
-----------
Inspected https://github.com/FreeRTOS/FreeRTOS/actions/runs/7496992648/job/20432412871?pr=1153. 

Checklist:
----------


- [ N/A ] I have tested my changes. No regression in existing tests.
  -  Gonna let the CI do this since I don't consider this change risky.
- [ N/A ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.
  -   No code changes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
